client/src/components/arm-components/cube-test-component.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useRef, useState } from 'react'
-import { useFrame } from 'react-three-fiber'
-import { useDrag } from "react-use-gesture"
+// import { useFrame } from 'react-three-fiber'
+// import { useDrag } from "react-use-gesture"
```

Comment:
IMO it is better just to delete code rather than commenting it out, unless we think there's a good chance someone will want to use this code again in the near future. Is that the case here?